front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -113,8 +114,10 @@ interface State {
   inputValue: string;
   focused: boolean;
   processing: boolean;
-  errorMessage: string | null;
```

Comment:
I don't remember the exact reason anymore, but there was some problem with keeping 1 error message in the state. So I replaced it with booleans, which made the error states easy to control.